ofl/abeezee/METADATA.pb

```diff
@@ -19,149 +19,150 @@ fonts {
-languages: "kde_Latn"  # Makonde
-languages: "kea_Latn"  # Kabuverdianu
-languages: "kg_Latn"  # Kongo
-languages: "kha_Latn"  # Khasi
-languages: "kj_Latn"  # Kuanyama
-languages: "kl_Latn"  # Kalaallisut
-languages: "kln_Latn"  # Kalenjin
-languages: "kmb_Latn"  # Kimbundu
-languages: "ksb_Latn"  # Shambala
-languages: "kw_Latn"  # Cornish
-languages: "la_Latn"  # Latin
-languages: "lb_Latn"  # Luxembourgish
-languages: "lij_Latn"  # Ligurian
-languages: "lmo_Latn"  # Lombard
-languages: "lua_Latn"  # Luba-Lulua
-languages: "luo_Latn"  # Luo
-languages: "luy_Latn"  # Luyia
-languages: "mfe_Latn"  # Morisyen
-languages: "mg_Latn"  # Malagasy
-languages: "mgh_Latn"  # Makhuwa-Meetto
-languages: "min_Latn"  # Minangkabau
-languages: "moe_Latn"  # Innu
-languages: "moh_Latn"  # Mohawk
-languages: "ms_Latn"  # Malay
-languages: "nap_Latn"  # Neapolitan
-languages: "nd_Latn"  # North Ndebele
-languages: "nds_Latn"  # Low German
-languages: "ng_Latn"  # Ndonga
-languages: "njo_Latn"  # Ao Naga
-languages: "no_Latn"  # Norwegian
-languages: "nov_Latn"  # Novial
-languages: "nr_Latn"  # South Ndebele
-languages: "nyn_Latn"  # Nyankole
-languages: "oc_Latn"  # Occitan
-languages: "om_Latn"  # Oromo
-languages: "pam_Latn"  # Pampanga
-languages: "pap_Latn"  # Papiamento
-languages: "pko_Latn"  # Pökoot
-languages: "pms_Latn"  # Piedmontese
-languages: "pon_Latn"  # Pohnpeian
-languages: "pt_Latn"  # Portuguese
-languages: "quc_Latn"  # Kʼicheʼ
-languages: "qug_Latn"  # Chimborazo Highland Quichua
-languages: "rm_Latn"  # Romansh
-languages: "rn_Latn"  # Rundi
-languages: "rof_Latn"  # Rombo
-languages: "rw_Latn"  # Kinyarwanda
-languages: "rwk_Latn"  # Rwa
-languages: "saq_Latn"  # Samburu
-languages: "sbp_Latn"  # Sangu
-languages: "sc_Latn"  # Sardinian
-languages: "scn_Latn"  # Sicilian
-languages: "sdc_Latn"  # Sassarese Sardinian
-languages: "seh_Latn"  # Sena
-languages: "sei_Latn"  # Seri
-languages: "sg_Latn"  # Sango
-languages: "sma_Latn"  # Southern Sami
-languages: "smj_Latn"  # Lule Sami
-languages: "sn_Latn"  # Shona
-languages: "snk_Latn"  # Soninke
-languages: "so_Latn"  # Somali
-languages: "sq_Latn"  # Albanian
-languages: "srn_Latn"  # Sranan Tongo
-languages: "ss_Latn"  # Swati
-languages: "su_Latn"  # Sundanese
-languages: "sv_Latn"  # Swedish
-languages: "sw_Latn"  # Swahili
-languages: "swb_Latn"  # Comorian, Latin
-languages: "teo_Latn"  # Teso
-languages: "tet_Latn"  # Tetum
-languages: "tiv_Latn"  # Tiv
-languages: "tpi_Latn"  # Tok Pisin
-languages: "ts_Latn"  # Tsonga
-languages: "tum_Latn"  # Tumbuka
-languages: "ug_Latn"  # Uyghur, Latin
-languages: "vmw_Latn"  # Makhuwa
-languages: "vo_Latn"  # Volapük
-languages: "vun_Latn"  # Vunjo
-languages: "wa_Latn"  # Walloon
-languages: "war_Latn"  # Waray
-languages: "wbp_Latn"  # Warlpiri
-languages: "xh_Latn"  # Xhosa
-languages: "xog_Latn"  # Soga
-languages: "yap_Latn"  # Yapese
-languages: "yua_Latn"  # Yucateco
-languages: "zu_Latn"  # Zulu
+languages: "aa_Latn"
```

Comment:
Packager is still removing the lang tag, until fix we should run gftools add-font manually on the family-dir

Reply:
@emmamarichal Marc just noted that we were using the last release of gftools, but the fix for this issue has not been released yet. If you install gftools with git+ it should work as expected.